### PR TITLE
fix: add userID from newUser entity && update context in login handler

### PR DESCRIPTION
**Pull request by voikin**
BUG `ASP-1280`

### CODE REVIEW

File: object/ldap_conn.go [hunk 1]
```diff
@@ -235,6 +240,8 @@ func SyncSingleUser(
 			return errors.New("LDAP sync error: failed to AddUser")
 		}
 
+		localUserID = newUser.Id
```
Comment (svgrafov) [inline object/ldap_conn.go:243]: Explicit is better than implicit. Please return ID from AddUser method.

If you dont want to fix AddUser() usages all across the repo, rename it to AddUserOld and make a copy of it with the name AddUser, returning ID.

File: object/ldap_conn.go [hunk 2]
```diff
@@ -191,6 +191,11 @@ func SyncSingleUser(
 	userExists := false
 	userExists = existingUserIDs[ldapUser.GetLdapUuid()]
 
+	localUserID, err := ldap_sync.GetLocalIDForExistingLdapUser(ldapUser.GetLdapUuid())
```
Comment (svgrafov) [inline object/ldap_conn.go:194]: Check GetLocalIDForExistingLdapUser() code. It returns error if no user is found. Which means you cant use it here.

**Review by svgrafov — CHANGES_REQUESTED** [review]

File: object/ldap_conn.go (hunk 1 above)
Comment (voikin) [reply object/ldap_conn.go]: done

File: object/ldap_conn.go (hunk 2 above)
Comment (voikin) [reply object/ldap_conn.go]: done

**Review by svgrafov — APPROVED** [review]

**Pull request merged**